### ISSUE TITLE
API-5, API-7: Epic for delivering MicroProfile 1.4 and 2.0

### ISSUE THREAD

**Issue by kwsutter** [OpenLiberty/open-liberty#4663]
This epic will be used to cover both MicroProfile 1.4 and 2.0:
https://bigblue.aha.io/features/API-5
https://bigblue.aha.io/features/API-7

MicroProfile 1.4 is based on Java EE 7 technologies, while MicroProfile 2.0 is based on Java EE 8 technologies.  The updated component features for both of these releases are the same and are tracked via their own features and epics.

- Rest Client 1.1, https://bigblue.aha.io/features/API-6, https://github.com/OpenLiberty/open-liberty/issues/4519
- Fault Tolerance 1.1, https://bigblue.aha.io/features/API-1, https://github.com/OpenLiberty/open-liberty/issues/2511
- Open Tracing 1.1, https://bigblue.aha.io/features/API-3, https://github.ibm.com/was-liberty/WS-CD-Open/issues/11723
- Config 1.3, https://bigblue.aha.io/features/API-2, https://github.com/OpenLiberty/open-liberty/issues/2508
- JWT 1.1, https://bigblue.aha.io/features/API-18, https://github.com/OpenLiberty/open-liberty/issues/4479

**Comment by kwsutter** [comment]
@gkwan-ibm Serviceability for MicroProfile is really covered by the component features that make up the MicroProfile platform.  If a user utilizes one of the microProfile-1.x or microProfile-2.0 convenience features, any problems will be associated with and directed to the respective component features that make up the platform.  I will still try to answer your questions...

1.  Yes, the WAD addresses the Serviceability question in the same light as my initial response above.

2. and 3.  I have asked the SVT team to test all flavors of the server.xml configuration when testing the respective component features.  For example, when testing the updates for mpJwt-1.1, the test coverage should utilize the standalone mpJwt-1.1 feature, as well as run the tests with the microProfile-1.4 and microProfile-2.0 features.

4.  The respective queues for each component will be utilized.  These top-level convenience features do not have individual support queues.

**Comment by kwsutter** [comment]
ID Issue:  https://github.ibm.com/was-liberty/liberty-docs/issues/377

**Comment by bsbyrd1** [comment]
My approval is based on the sizing of the ID work, and the expectation that the ID work can be completed before the last ID translation drop.

**Comment by dave-waddling** [comment]
**FAT Focal Approval:** Please can you complete the Feature Test Summary issue (#4754) that has been added to your Epic. This issue is what is used to review the FAT testing and obtain the FAT focal approval. This feature pulls together the delivery of a number of other features, each of which will have their own FAT focal approval, so please use this Feature Test Summary to describe any additional testing which isn't covered in the epics listed in the description. Many thanks!

**Comment by dave-waddling** [comment]
**FAT Focal Approval:** Given that JWT (#4479) is not going to be approved for 3Q due to a TCK not being in automation I don't believe it will be possible to grant a FAT Focal Approval for this epic given that it covers the delivery of that work.

**Comment by kwsutter** [comment]
@dave-waddling I heard from @chunlongliang-ibm that we can still ship JWT with a manual run of the TCK, so we should be back in business for a 3Q delivery for both JWT and these platform, convenience MicroProfile features.  I will complete your FAT issue #4754 .

**Comment by dave-waddling** [comment]
@kwsutter That's great news, thank you! I've read through the Feature Test Summary and it's as I'd expected for this Epic. A couple of the other related features still need to get their approvals through so this Epic should end up getting approved on Friday (2018-08-24);I'm agreeable to approving this Epic even though JWT won't be approved (it's good enough that the JWT Epic remains open to ensure the delivery of the TCK automation).

**Comment by kwsutter** [comment]
WAD link (may not be accessible outside of IBM):  https://ibm.box.com/s/av9dgzce0k2qmgz3h7jgnzmak7r8hlhs

**Comment by dave-waddling** [comment]
**FAT Focal Approval:** Granting approval as per the call.

**Comment by kwsutter** [comment]
All approvals in place and functionality delivered!

**Comment by marikaj123** [comment]
Blog GA Info: https://github.ibm.com/was-liberty/WS-CD-Open/issues/13133

**Comment by marikaj123** [comment]
Blog Beta9 Info: https://github.ibm.com/was-liberty/WS-CD-Open/issues/13137
